app/src/components/largeButton.js

```diff
@@ -0,0 +1,5 @@
+export default function LargeButton({text}) {
+  return (
+    <button className="bg-blue-500 hover:bg-blue-700 border border-blue-500 text-white font-bold py-3 px-6 rounded-lg text-4xl">{text}</button>
+  );
+}
```

Comment:
Should I create a new class like this? Or should we just use the regular button class and allow us to specify text size and button size in the props?